autonomy/test_tools/configurations.py

```diff
@@ -31,8 +31,6 @@ def get_key(key_path: Path) -> str:
 
 CUR_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))  # type: ignore
 TEST_TOOLS_DIR = Path(CUR_PATH).resolve().absolute()
-ROOT_DIR = Path(CUR_PATH, "..", "..").resolve().absolute()
-THIRD_PARTY = ROOT_DIR / "third_party"
```

Comment:
so now we always obtain it the THIRD_PARTY as a constant from the conftest.py file in the respective test directory of whatever repository autonomy is installed in, am I understanding correctly?

this then creates a constraint on any repository using the autonomy package to specify these in their `tests/conftest.py` file?

Reply:
Not necessarily from the conftest.py, you can set it on the test class like this

https://github.com/valory-xyz/open-autonomy/pull/1199/files#diff-7dbf9570300e3300f3e8c673ea8087d701d596adf755023c02b5035a10c9a803R75

Reply:
So now we should provide the necessary contracts ourselves in our repository. Would it be an option to link the respective contracts as submodules along with the package?

for example, `GnosisSafeNetDockerImage` replies on `safe-contracts` specifically.